### DISCUSSION TITLE
Index Optimize jobs question

### DISCUSSION THREAD

**Discussion by craig960** [olahallengren/sql-server-maintenance-solution#615] [Questions & Answers] [answered]
Hi @olahallengren 

With your solution, for nightly index/stats maintenance I was wondering is it better to have 1 job and pass in all the dbs you want to perform index maintenance on, or is it better to create multiple jobs with each just looking at one db.  I would like to be able to run the index maintenance on multiple dbs in parallel so that they can all complete overnight in a 6 hour time window.  If I was to do them sequentially it would take too long to complete.

**Comment by jacobgexigo** [comment, marked as answer]
Make sure you are using the newest version of the solution and use the "DatabasesInParallel" parameter to let it do the work in parallel using the queue table to keep track.

![image](https://user-images.githubusercontent.com/29784949/159706692-c25dba39-c53b-47f3-81ec-64513d4185a2.png)

**Reply by craig960** [reply]
Thanks, yes I found this after I posted, and it is working perfectly.
